e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -52,14 +55,30 @@ public void When_ExtractRequestReceived_Expect_ExtractStatusAndDocumentDataAdded
         assertThatExtractCoreMessageWasSent(ehrExtractCore);
     }
 
-    private void assertThatInitialRecordWasCreated(String conversationId, Document ehrExtractStatus) {
+    @Test
+    public void When_ExtractRequestReceivedForPatientWithNoDocs_Expect_DatabaseToBeUpdatedAccordingly() throws Exception {
+        String conversationId = UUID.randomUUID().toString();
```

Comment:
We could use a hard coded conversation id for the testing purposes.

Reply:
No, it needs to be random. The database is not cleared down for every test.

Reply:
Sounds like something which could be implemented. Leaving data in the database after testing is not a good practise.

Reply:
Usually it is undesirable, but in this case is needed. Spine uses this ConversationId to track the transaction, so it should not be reused for multiple transactions. We will need to run these tests in shared environments where we don't want to clear the database. All records are deleted automatically after 7 days so we don't need to worry about filling up the database with random test data.

Reply:
Ok